openc3/data/config/target_config.yaml

```diff
@@ -13,10 +13,7 @@ LANGUAGE:
   since: 5.11.1
 REQUIRE:
   summary: Requires a Ruby file
-  description:
-    Ruby files must be required to be available to call in other code.
-    Files are first required from the target's lib folder. If no file is found the
-    Ruby system path is checked which includes the base openc3/lib folder.
+  description: Ruby files optionally be required to explicitly declare dependencies.
```

Comment:
It's optional. I'd be ok removing it but I guess it's interesting to explicitly declare dependencies.